openvdb/openvdb/tree/ValueAccessor.h

```diff
@@ -1471,16 +1888,15 @@ class ValueAccessor1 : public ValueAccessorBase<_TreeType, IsSafe>
     {
         assert(BaseT::mTree);
         static_assert(!BaseT::IsConstTree, "can't get a non-const node from a const tree");
-        OPENVDB_NO_UNREACHABLE_CODE_WARNING_BEGIN
         if ((std::is_same<NodeT, NodeT0>::value)) {
             if (this->isHashed(xyz)) {
                 assert(mNode0);
                 return reinterpret_cast<NodeT*>(const_cast<NodeT0*>(mNode0));
             }
             return BaseT::mTree->root().template probeNodeAndCache<NodeT>(xyz, *this);
         }
-        return nullptr;
-        OPENVDB_NO_UNREACHABLE_CODE_WARNING_END
+        // No point trying to cache as this NodeT isn't part of this accessors chain
+        return BaseT::mTree->root().template probeNode<NodeT>(xyz);
```

Comment:
Bugfix - even if `NodeT` isn't being cached it might exist and a nullptr will always be returned